src/palace/manager/feed/acquisition.py

```diff
@@ -608,7 +605,7 @@ def single_entry_loans_feed(
     @classmethod
     def single_entry(
         cls,
-        work: Work | Edition | None,
+        work: Work | Edition,
```

Comment:
I looked at everyone calling this function to make sure they are not (or updated them) passing `None` here.